pkg/cloud/cloud_test.go

```diff
@@ -3031,6 +3031,7 @@ func TestWaitForAttachmentState(t *testing.T) {
 			switch tc.name {
 			case "success: detached", "failure: already assigned but wrong state":
 				mockEC2.EXPECT().DescribeVolumes(gomock.Any(), gomock.Any()).Return(&ec2.DescribeVolumesOutput{Volumes: []types.Volume{detachedVol}}, nil).AnyTimes()
+				mockEC2.EXPECT().AttachVolume(gomock.Any(), gomock.Any()).Return(nil, nil).AnyTimes()
```

Comment:
can we move this into it's own case, and drop the `AnyTimes`? because `AnyTimes` means that _zero_ times is accepted

Reply:
Done ✅